### PR TITLE
feat(plugins): use `@tauri-apps` namespace on NPM

### PR DESCRIPTION
**Pull request by lucasfernog**
Initially I wanted to use `@tauri-apps` but the conflict between the tauri CLI `@tauri-apps/cli` and the CLI plugin name sucks.

### CODE REVIEW

**Comment by amrbashir** [comment]
I think we should instead be using `@tauri-apps/plugin-*` so it feels familiar and more authentic and if we ever move to `@tauri/*` we can still have the CLI and the CLI plugin under the same org.

File: plugins/dialog/README.md [hunk 1]
```diff
@@ -18,19 +18,19 @@ Install the Core plugin by adding the following to your `Cargo.toml` file:
 
 ```toml
 [dependencies]
-tauri-plugin-dialog-api = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
+@tauri-apps/plugin-dialog = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
```
Comment (amrbashir) [inline plugins/dialog/README.md]:
```suggestion
tauri-plugin-dialog = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
```

File: plugins/fs-watch/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tauri-plugin-fs-watch-api",
+  "name": "@tauri-apps/plugin-fs-watch",
```
Comment (amrbashir) [inline plugins/fs-watch/package.json:2]: could we just remove `fs-wtach` and `fs-extra` plugins and merge them to `fs` plugin instead?

**Review by amrbashir — CHANGES_REQUESTED** [review]

File: plugins/fs-watch/package.json (hunk 2 above)
Comment (lucasfernog) [reply plugins/fs-watch/package.json:2]: `fs-extra` was removed already, but there's some documentation to fix.

Comment (lucasfernog) [reply plugins/fs-watch/package.json:2]: and yeah `fs-watch` should be on `fs` under a feature flag. We can do that later.

Comment (amrbashir) [reply plugins/fs-watch/package.json:2]: definitely before release, seems redundant to release it now and remove it later.

**Review by amrbashir — APPROVED** [review]

File: plugins/fs-watch/package.json (hunk 2 above)
Comment (lucasfernog) [reply plugins/fs-watch/package.json:2]: I'll work on it.

**Pull request merged**